_commands/basics/echo.md

```diff
@@ -3,22 +3,71 @@
 
 echo
 ---
-TODO: Add documentation for this command by submitting a pull request.
+
+`echo` is used to display message on screen, and write given string to standard output, with each a space between each and a newline after the last one.  `echo` is implemented as builtin command in many command shells like bash, ksh, csh.
```

Comment:
There are a few grammatical errors in this sentence. I've provided a cleaned up version here:
`echo` is used to display a message on the screen and write the given message to standard output with a newline at the end.  `echo` is implemented as a built-in command in many shells like bash, ksh, and csh.